### DISCUSSION TITLE
Welcome to Neural-Network Discussions!

### DISCUSSION THREAD

**Discussion by jerryjee120** [jerryjee120/Neural-Network#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Grudan2022** [comment]
Hi Jerry.  Really nice work.  I'm wondering, is there a way to use this with my own dataset?

**Reply by jerryjee120** [reply]
Yes, you can. 
1.  the model you created can be used for your own dataset; 
2. you can use your dataset with an existing model, but you may need to do some encapsulation of your dataset;
3. you can also try something in the "Model/Train"；
4. the tutorials contains a lot of source code and examples, you can replace it with your own dataset according to the situation.

You can come up with more specific ideas and needs. We will continue to improve.
For specific discussions, you can create an issue or discussion.

Thanks for you feedback!

**Comment by chenlinhui1998** [comment]
hello！I am very interested in your neural network visualization! Can I get an in-depth understanding with you? We can contact by email! 634805385@qq.com

**Reply by chenlinhui1998** [reply]
Maybe there will be extra gains!

**Comment by friday-chen** [comment]
I think this project is incredible! I am curious, was this project implemented through on-device training via tf lite?
